.VERSION

```diff
@@ -0,0 +1 @@
+v0.6.0-311-gf435e33b
```

Comment:
Is this something we should commit?

Reply:
It should probably be 'local', but as it's a compile time resource it's fiddly to default in a macro. Has to be done externally to cargo, and I want to keep cargo build (not just make) working out of the box as much as possible